api/R/submit_workflow.R

```diff
@@ -59,3 +65,47 @@ submit_workflow <- function(settings,
   message("Follow workflow progress from your browser at:\n", follow_url)
   httr::content(result)
 }
+
+#' Convert List to XML
+#'
+#' Can convert list or other object to an xml object using xmlNode
+#' @param item object to be converted. Despite the function name, need not actually be a list
+#' @param tag xml tag
+#' @return xmlNode
+#' @author David LeBauer, Carl Davidson, Rob Kooper
+listToXML <- function(item, tag) {
```

Comment:
I basically copied this code from `PEcAn.settings`. That does mean it's duplicated. However, I think this minor duplication of code that hasn't changed in ages is justified to avoid needing to pull in all of `PEcAn.settings`.

Reply:
I agree, this would be a nice function for PEcAn.utils if it only container helper functions like this.